### ISSUE TITLE
[🐛BUG] Preload embedding doesn't work

### ISSUE THREAD

**Issue by MochizukiShinichi** [RUCAIBox/RecBole#1721]
**描述这个 bug**
I used WideDeep recommender and successfully loaded customized user embedding following [this](https://recbole.io/docs/user_guide/usage/load_pretrained_embedding.html) tutorial but Recbole still creates dimension = [#user_id, #embedding_size] embedding layer in the code.

**如何复现**
config
```
"additional_feat_suffix": ["useremb", ],

'load_col': {'inter': ['unique_id', 'item_id', "rating"],
            "useremb": ["uid", "user_emb"]},

"alias_of_user_id": ["uid"],
"preload_weight": {"uid": "user_emb"},
    
"USER_ID_FIELD": "unique_id",
"ITEM_ID_FIELD": "item_id",
"RATING_FIELD": "rating",
```

**实验环境（请补全下列信息）：
- RecBole 版本: 1.1.1
 - Python 版本: 3.8

**Comment by peteryang1031** [comment]
Hi @MochizukiShinichi, thank you for your attention to Recbole! The problem may occur when you have not manually replace the original embedding with your pretrained embedding in the model structure. It is suggested to try it again and follow the last entry in your mentioned [tutorial](https://recbole.io/docs/user_guide/usage/load_pretrained_embedding.html).

**Comment by MochizukiShinichi** [comment]
Hi @peteryang1031, thank you very much for your reply. Does that mean existing model structures don’t support customized embedding and users would need write customized model?

**Comment by MochizukiShinichi** [comment]
Hi @peteryang1031  thank you again for your comments. I found [this](https://github.com/RUCAIBox/RecBole/issues/1150) duplicated issue and I tried to subclass a model I wanted like this and it seems to be working fine.
Is this the right approach? 
```from recbole.model.context_aware_recommender import WideDeep
from torch import nn

class WideDeepCustom(WideDeep):
    def __init__(self, config, dataset):
        super(WideDeepCustom, self).__init__(config, dataset)
        pretrained_user_emb = dataset.get_preload_weight('uid')
        self.user_embedding = nn.Embedding.from_pretrained(torch.from_numpy(pretrained_user_emb))
```

**Comment by peteryang1031** [comment]
Yeah, using subclass is a highly effective way to load pretrained embeddings. As [#1150](https://github.com/RUCAIBox/RecBole/issues/1150) mentioned, modification is needed in Recbole when using pretrained embeddings.
